pydantic/_internal/_generate_schema.py

```diff
@@ -532,8 +532,10 @@ def _common_field_schema(self, name: str, field_info: FieldInfo, decorators: Dec
         return _common_field(
             schema,
             serialization_exclude=True if field_info.exclude else None,
-            validation_alias=field_info.alias,
-            serialization_alias=field_info.alias,
+            validation_alias=field_info.validation_alias,
+            serialization_alias=(
```

Comment:
better that this is just `serialization_alias=field_info.serialization_alias`, then change field.

Reply:
done